test/blocks/global-navigation/utilities/utilities.test.js

```diff
@@ -156,4 +158,27 @@ describe('global navigation utilities', () => {
     config.imsClientId = ogImsClientId;
     setConfig(config);
   });
+
+  describe('LANA logs', () => {
+    it('should send LANA log on error', async () => {
+      // Mock the global window.lana.log method
+      const originalLanaLog = window.lana.log;
+      const lanaLogSpy = sinon.spy();
+      window.lana.log = lanaLogSpy;
+
+      // The function that will throw an error.
+      const erroneousFunction = async () => {
+        throw new Error('error');
+      };
+
+      // Call logErrorFor.
+      await logErrorFor(erroneousFunction, 'message', 'someTags');
+
+      // Check if lanaLog (through window.lana.log) was called with expected parameters.
+      expect(lanaLogSpy.calledOnce).to.be.true;
```

Comment:
You could expand the check to see that the tags and messages are also correctly called